src/tools/contrib/lscpu.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+"""
+This module gets the output from lscpu and produces a list of CPU uids
+corresponding to physical cores.
```

Comment:
ditto, can you please separate the commits?

Reply:
I'll try splitting as per https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History 🤞